packages/client/lib/rpc/modules/eth.ts

```diff
@@ -459,23 +461,25 @@ export class Eth {
     const vm = this._vm.copy()
```

Comment:
If we copy the VM we still save the `runCall` stuff to disk right? (We discussed this yesterday on the call?). If VM originally wrote things to disk then it would also do this time right?

Reply:
It copies the StateManager, which copies the trie, so it shouldn't write it to the original trie.